DataConnectors/CEF/cef_AMA_troubleshoot.py

```diff
@@ -697,13 +701,27 @@ def verify_root_privileges(printer):
             "This script must be run in elevated privileges since some of the tests require root privileges")
         exit()
 
+def find_dcr_cloud_environment():
+    '''
+    Use the agent config on the machine to determine the cloud environment we are running in
+    '''
+    global MACHINE_ENV
+    command_to_run = "if grep -q \"azure.com\" {0}; then echo \"Prod\"; elif grep -q \"azure.cn\" {0}; then echo \"MK\"; elif grep -q \"azure.us\" {0}; then echo \"Gov\"; else echo \"Prod\"; fi".format(AGENT_CONF_FILE)
```

Comment:
I think this will be simpler
"if grep -q \"azure.cn\" {0}; then echo \"MK\"; elif grep -q \"azure.us\" {0}; then echo \"Gov\"; else echo \"Prod\"; fi".format(AGENT_CONF_FILE)

Reply:
Done.